src/ctask.jl

```diff
@@ -33,11 +33,11 @@ proper way is refreshing the `current_task` (the variable `t`) in
 `start_task` after the call to `jl_apply` returns.
 
 """
-function task_wrapper(func)
+function task_wrapper(func; _cow=true)
```

Comment:
maybe keep the keyword argument consistent with `CTask`?

```suggestion
function task_wrapper(func; cow=true)
```

Reply:
If we use `cow` in the arguments, it will shadow the function `cow`. 

Now I change the argument to `cow` and the function to `_cow`.